packages/native-stack/src/types.tsx

```diff
@@ -439,6 +447,13 @@ export type NativeStackNavigationOptions = {
    * Style object for the scene content.
    */
   contentStyle?: StyleProp<ViewStyle>;
+  /**
+   * Style object for the screen native component. This might help to workaround
+   * some issues when using `formSheet` presentation.
+   *
+   * Only `backgroundColor` is accepted.
+   */
+  unstable_screenStyle?: ScreenProps['unstable_screenStyle'];
```

Comment:
This is needed right now to workaround "truncated formsheet" issue on iOS, where due to asynchronous nature of react's layout we do not trigger the layout when sheet height changes, causing view height to not be updated and effectively truncated in case its height is less than sheet height.

To workaround this, the user can set background color of the screen component itself. If I recall correctly it can not be set through `contentStyle`, cause the view the `contentStyle` is set on is also truncated. 

> [!note]
It might be worth considering to just hijack the value of `contentStyle.backgroundColor` and pass it to Screen component in case formSheet presentation is in use. Idk whether that is feasible, but if so, that might be the best solution.